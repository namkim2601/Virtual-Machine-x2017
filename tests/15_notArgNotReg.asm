FUNC LABEL 0
    MOV REG 0 VAL 8
    MOV STK A REG 0
    NOT STK A
    RET
