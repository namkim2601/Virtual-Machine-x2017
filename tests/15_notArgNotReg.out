Error: OPCODE 'NOT' argument must be a register
